src/style.css

```diff
@@ -46,13 +58,21 @@ form {
 button {
   height: 1.5rem;
   width: 4rem;
-  border: 2px solid #000;
+  border: 2px solid #0541a9;
+  background-color: #e1cdaf;
+  border-radius: 4px;
+  color: #0541a9;
+  box-shadow: 10px 5px 5px #000;
 }
```

Comment:
- [OPTIONAL] I suggest adding hover effects for your buttons to improve the user experience :recycle: